Model/Media.php

```diff
@@ -643,8 +643,11 @@ public function getCategory()
     /**
      * @param CategoryInterface $category|null
      */
-    public function setCategory(CategoryInterface $category = null)
+    public function setCategory($category = null)
```

Comment:
This is true if someone create his own Media class, but the BIG problem has been add the strong dependency with classification bundle that it is a feaure for a media bundle but not a requirement.
I haven't known other solution that break the BC and fix the real problem.